docs/architecture/stage_list.md

```diff
@@ -0,0 +1,1060 @@
+---
+---
+
+Stage list
+----------
```

Comment:
I am under impression that html page I used as original for this MD file was generated from wiki page, which was in turn generated (as MD) by some SomethingSomethingGenerator or other @rvansa created some time ago. The plan to rewrite this to generate MD directly into page source is in the pipe, intended end-game for this and traits-list too. Also, I have this notion of making this more searchable/interactive, probably via some JS plugin, but this is in the wishful thinking stage presently.